src/ovh-ui-angular/decisions.md

```diff
@@ -6,11 +6,13 @@
 4. Use `id` and `name` attributes on form components
 5. Keep translation outside this library
 6. Components
-    1. Expose only form events that are really used
-    2. Use the `text` attribute for component inner text without html inside
-    3. Be more restrictive on component attributes and signature
-    4. Throw warning when concurrent modifiers are used at the same time
-    5. Throw warning when aria text are missing
+    1. For now, keep component code as simple as possible
+    2. Expose only form events that are really used
+    3. Use the `text` attribute for component inner text without html inside
+    4. Group together states that are exclusive in a single attribute
+    5. Be more restrictive on component attributes and signature
+    6. Throw warning when concurrent modifiers are used at the same time
+    7. Throw warning when aria text are missing
```

Comment:
Maybe we should let an external library do that ? 
To keep the focus on what the component should really do.

Reply:
We can talk more about that idea later and when we will really start doing `aria` things.